The Economic Times.js

```diff
@@ -0,0 +1,192 @@
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2017 Sonali Gupta
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.indexOf("/topic/") != -1) {
+		return "multiple";
+	}
+	else if (ZU.xpathText(doc, '//article')) {
+		return "newspaperArticle";
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//main//a[(h2 or h3) and contains(@href, "/articleshow")]');
+	for (var i=0; i<rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function scrape(doc, url) {
+	
+	newItem = new Zotero.Item("newspaperArticle");
+	newItem.url = url;
+	newItem.publicationTitle = "The Economic Times";
+
+	//get headline
+	var title = ZU.xpathText(doc, '//article/h1');
+	if (!title) title = ZU.xpathText(doc, '//meta[@property="og:title"]/@content');
+	newItem.title = title;
+
+	//get abstract
+	var abstract = ZU.xpathText(doc, '//meta[@property="og:description"]/@content');
+	newItem.abstractNote = abstract;
+	
+	//get date
+	var xpathdate_author = '//article/div/div[contains(@class, "publish_on") or contains(@class, "byline")]';
+	var date = ZU.xpathText(doc, xpathdate_author);
+	if (date) {
+		newItem.date = ZU.strToISO(date);
+	}
+
+	//get author or organization
+	var authors = ZU.xpath(doc, '//a[@rel="author"]');
+	for (var i in authors){
+		newItem.creators.push(ZU.cleanAuthor(authors[i].textContent, "author"));
+	}
+	if(!authors.length){
+		authors = ZU.xpathText(doc, xpathdate_author);
+		if(authors){
+			authors_org=authors.substring(0,authors.lastIndexOf("|")-1);
+			var regex = /(.*By\s+)(.*)/;
+			authors = authors_org.replace(regex, "$2");
+			newItem.creators.push({lastName:authors,  creatorType: "author",fieldMode: 1})
```

Comment:
Add space before `fieldMode`, delete one of the two spaces before `creatorType`.